pennylane/qcut/tapes.py

```diff
@@ -102,9 +102,9 @@ def tape_to_graph(tape: QuantumTape) -> MultiDiGraph:
 
 
 # pylint: disable=protected-access
-def graph_to_tape(graph: MultiDiGraph) -> QuantumTape:
+def graph_to_tape(graph: MultiDiGraph) -> QuantumScript:
```

Comment:
We leave publicly documented functions as `QuantumTape` even if it always returns a `QuantumScript`, see the [recent doc on it](https://docs.pennylane.ai/en/latest/code/qml_tape.html) ("Since users are familiar with the term QuantumTape, that term should be used in documentation.")
```suggestion
def graph_to_tape(graph: MultiDiGraph) -> QuantumTape:
```